test/emoji.ref

```diff
@@ -9,35 +9,31 @@
     <h1>This is a test</h1>
     <div class="admonition admonition-note alert alert--secondary">
       <div class="admonition-heading">
-        <h5>
-          <div class="admonition-icon">ℹ️</div>note</h5>
+        <h5><span class="admonition-icon">ℹ️</span>note</h5>
```

Comment:
Because the tests are really flaky the difference in whitespace causes them to fail.